jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "introduction to the CircleCI API v2"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 is a significant update to the existing CircleCI API v1.1 that is currently being used by internal and external audiences.
+
+## Introduction to API v2
+
+CircleCI API v2 has significantly changed the current API v1.1 paradigm to incorporate some important changes and shifts in the way that you can and will use the API for your jobs. Although changes were made to the available endpoints (discussed later on this page), two of the most important changes are the introduction of "Pipelines" and "Project Slugs" as concepts in the API design and architecture, as well as a set of new endpoints that enable you to:
```

Comment:
One tone question: generally docs have been written to describe current state rather than historical context. A newcomer to this document won't have any context about 1.1, so this is written only to an existing 1.1 user. I recommend we move the "paradigm" change to a page specifically about the delta from 1.1 and keep the intro page in a neutral tone that describes v2 as if it is The API.

Reply:
If you DO keep this here, then I'd change the title to "What's new in API v2?" rather than "Introduction", though personally I'd recommend moving that to a separate page, as it will become increasingly obsolete to talk about v2 in terms of 1.1.

Reply:
Just to +1 @ndintenfass comments. An example I can point to is the [GitHub v3](https://developer.github.com/v3/) and [GitHub v4](https://developer.github.com/v4/) APIs. While the docs are generally aware of each other, they both have neutral tones and push aside conversations about migrating to the newer APIs in a separate (document)[https://developer.github.com/v4/guides/migrating-from-rest/]

Reply:
Fixed by removing content that distinguishes features of 1.1 and 2